packages/date/src/utils/timezones.ts

```diff
@@ -0,0 +1,20 @@
+import dayjs from 'dayjs';
+
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+export type Timezone = {
+  displayValue: string;
+  ianaName: string;
+};
+
+//@ts-expect-error
+const timezones = Intl.supportedValuesOf('timeZone').map((timezone: string) => ({
```

Comment:
The support for `Intl.supportedValueOf` is almost non-existent 😞 

![image](https://user-images.githubusercontent.com/12673605/168570566-d35d62c6-5478-43aa-8667-b5198d425e85.png)

Reply:
Yeah, I think we don't have a better option. 25kb is better than 100kb 🤷🏻‍♂️ . I'm okay with using that package (a better option would be finding another regularly updated package that gives us only the names - nothing extra - that would save a few kbs)

But after you get the list of TZ names remember to check each if it's supported. We can borrow [some logic from this polyfill](https://github.com/tc39/proposal-intl-enumeration/blob/e09f83795e4dfc4c3d8ad2804f81f7396b7125ae/polyfill/timeZones.js#L100-L110) for that.

Reply:
How does it look to you now?